github/ci/prow/files/config.yaml

```diff
@@ -29,8 +29,8 @@ sinker:
   max_prowjob_age: 48h
   max_pod_age: 30m
 
-prowjob_namespace: kubevirt-prow
-pod_namespace: kubevirt-prow
+prowjob_namespace: kubevirt-prow-jobs
+pod_namespace: kubevirt-prow-jobs
```

Comment:
wasn't the idea to use different namespaces? :thinking:

Reply:
Haha, for now different than the prow components.

Reply:
got it ;) :+1: